static/js/modules/ckeditor.wysiwyg.js

```diff
@@ -1,7 +1,11 @@
 /* Default CKEDITOR Styles
  * Added within src/settings.py CKEDITOR_SETTINGS.stylesSet
  */
+
+/* allow empty span */
+CKEDITOR.dtd.$removeEmpty.span = 0;
```

Comment:
Please add it at the very button with the following style:

```
// extend ckeditor default settings
// DOCS: http://docs.ckeditor.com/#!/api/CKEDITOR.dtd
CKEDITOR.dtd.$removeEmpty.span = 0;
```

- note it on issue #57 
- adapt the CHANGELOG.txt accordingly